classes/02class/exercises/c02-network05/ramiray/README.md

```diff
@@ -0,0 +1,58 @@
+# c02-network05
+
+## Commands Execution Output
+
+- Commands for creating the internet gateway:
+```
+## Creating IGW:
+
+aws ec2 create-internet-gateway
+
+aws ec2 create-internet-gateway
+{
+    "InternetGateway": {
+        "Tags": [], 
+        "Attachments": [], 
+        "InternetGatewayId": "igw-0b50e51e2cf01b373"
+    }
+}
+
+## Naming the igw as "devopsacademy-igw":
+
+aws ec2 create-tags --resources igw-0b50e51e2cf01b373 --tags Key=Name,Value=devopsacademy-igw
```

Comment:
Alternatively, you can do add tags in the same command that you create:

```bash
aws ec2 create-internet-gateway \
  --tag-specifications "ResourceType=internet-gateway,Tags=[{Key=Name,Value=devopsacademy-igw}]"
```
Check the [Create Internet Gateway](https://docs.aws.amazon.com/cli/latest/reference/ec2/create-internet-gateway.html#options) from docs for reference.
You also can see the same help from the CLI with `aws ec2 create-internet-gateway help`.